test/unit-test/cellular_at_core_utest.c

```diff
@@ -938,19 +934,17 @@ void test_Cellular_ATStrDup_Invalid_Param( void )
 {
     CellularATError_t cellularStatus = CELLULAR_AT_SUCCESS;
     char * pDst;
-    char * pSrc;
+    char pSrc[ sizeof( CELLULAR_SAMPLE_PREFIX_STRING_LARGE_INPUT ) ] = { 0 };
```

Comment:
To be equivalent this would have to be CELLULAR_SAMPLE_PREFIX_STRING_LARGE_INPUT+1 ?

Reply:
CELLULAR_SAMPLE_PREFIX_STRING_LARGE_INPUT is an character string array pre-defined in Macro.
The array of characters terminated by a null character '\0', and this character won't be counted by strlen() but sizeof().
So sizeof(arrary) equal to strlen(array) + 1.